tools/crm_resource.c

```diff
@@ -1739,6 +1739,13 @@ main(int argc, char **argv)
                         "Resource '%s' not found", options.rsc_id);
             goto done;
         }
+
+        if (strchr(options.rsc_id, ':') != NULL && pe_rsc_is_clone(rsc->parent)) {
+            exit_code = CRM_EX_INVALID_PARAM;
+            g_set_error(&error, PCMK__EXITC_ERROR, exit_code,
+                        "Cannot operate on clone resource instance '%s'", options.rsc_id);
+            goto done;
+        }
```

Comment:
I think we only want to do this for --restart, --move, --ban, --clear. Instance numbers are reasonable for other commands

Reply:
Updated the last comment to check command and also reinstated the CTS tests.